README.md

```diff
@@ -142,6 +142,12 @@ You can generate an update site using [Maven](http://maven.apache.org/):
 This will create an update site in
 _com.github.eclipsecolortheme.updatesite/target/repository_.
 
+By default, Eclipse Color Theme will be built against the Eclipse Mars
+version of the platform. You can build against either Eclipse Luna or
+Kepler by setting the "platform-version" property, for example:
+
+    mvn clean verify -Dplatform-version=kepler
+
 License
```

Comment:
Users just need stable plugins. Having one version that works within 4.3-4.5 is simpler.

Why to have build against Mars?